packages/python/accern_xyme/accern_xyme.py

```diff
@@ -1142,8 +1145,14 @@ def get_company(self) -> str:
     def get_state_type(self) -> str:
```

Comment:
rename to get_state_uri

Reply:
ok

Reply:
I see it just wasn't pushed yet? okay nvmd then -- was just confusing for me -- but I can wait until you push :)